packages/preview/outrageous/0.1.0/typst.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "outrageous"
+version = "0.1.0"
+authors = ["RubixDev"]
+entrypoint = "outrageous.typ"
+keywords = ["outline"]
+license = "GPL-3.0-only"
+repository = "https://github.com/RubixDev/typst-outrageous"
+description = "Easier customization of outline entries."
```

Comment:
could you exclude the example files with the `exclude` key to prevent the package size from being unnecessarily large?

Reply:
done